### DISCUSSION TITLE
http://" + ip_port,

### DISCUSSION THREAD

**Discussion by V4rial** [KevinBytesTheDust/CVAmp#300] [General]
Hey bud, nice one, I would like to suggest to use:
                self.proxy_list.append(
                    {
                        "server": "http://" + ip_port,
                    }
                )
instead of user and password + ip_port.. 
modified it to work like that, works like a charm..
since thats not my work, i didnt post anywhere, just a suggestion ;)

also i added to work using http requests, viewers are rising faster than a hurricane lol

**Comment by KevinBytesTheDust** [comment]
Thank you for the suggestions. You can use the format `ip:port::` if your proxies do not have auth.
